modules/ROOT/pages/example-einstein-prompts.adoc

```diff
@@ -0,0 +1,102 @@
+= Example Einstein Prompts 
+
+Following are some use cases and example Einstein prompts for each situation. These prompts were created based on the sample data used during each test and are meant to provide insight about creating prompts to analyze and format your custom documents. 
```

Comment:
```suggestion
Following are some use cases and example Einstein prompts for each situation. These prompts are based on the sample data used during each test and are meant to provide insight about creating prompts to analyze and format your custom documents. 
```

Reply:
You might also consider replacing "are meant to provide insight about creating" with a simpler like "show how to create".